charts/zabbix/templates/deployment-zabbix-server.yaml

```diff
@@ -63,60 +63,55 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- if gt (.Values.zabbixserver.replicaCount | int) 1 }}
       initContainers:
+      {{- if gt (.Values.zabbixServer.replicaCount | int) 1 }}
         - name: init-wait-for-database-schema
-          {{- if .Values.zabbixserver.image.tag }}
-          image: "{{ .Values.zabbixserver.image.repository }}:{{ .Values.zabbixserver.image.tag }}"
+          {{- if .Values.zabbixServer.image.tag }}
+          image: "{{ .Values.zabbixServer.image.repository }}:{{ .Values.zabbixServer.image.tag }}"
           {{- else }}
-          image: "{{ .Values.zabbixserver.image.repository }}:{{ .Values.zabbix_image_tag }}"
+          image: "{{ .Values.zabbixServer.image.repository }}:{{ .Values.zabbixImageTag }}"
           {{- end }}
           env:
-            {{- include "zabbix.db_access.env_vars" (list $ . "zabbix") | nindent 12 }}
+            {{- include "zabbix.postgresAccess.variables" (list $ . "zabbix") | nindent 12 }}
           command:
             - "/bin/bash"
             - "/script/wait_db_schema.sh"
           volumeMounts:
             - name: init-waitschema-script
               mountPath: /script
-        {{- with .Values.zabbixserver.extraInitContainers }}
-        {{- toYaml . | nindent 8 }}
-        {{- end }}
-      {{- else if eq (.Values.zabbixserver.replicaCount | int) 1 }}
-      initContainers:
-        {{- with .Values.zabbixserver.extraInitContainers }}
+      {{- end }}
+        {{- with .Values.zabbixServer.extraInitContainers }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
-      {{- end }}
       containers:
         - name: zabbix-server
           resources:
-{{- toYaml .Values.zabbixserver.resources | nindent 12 }}
-          {{- if .Values.zabbixserver.image.tag }}
-          image: "{{ .Values.zabbixserver.image.repository }}:{{ .Values.zabbixserver.image.tag }}"
+{{- toYaml .Values.zabbixServer.resources | nindent 12 }}
+          {{- if .Values.zabbixServer.image.tag }}
+          image: "{{ .Values.zabbixServer.image.repository }}:{{ .Values.zabbixServer.image.tag }}"
           {{- else }}
-          image: "{{ .Values.zabbixserver.image.repository }}:{{ .Values.zabbix_image_tag }}"
+          image: "{{ .Values.zabbixServer.image.repository }}:{{ .Values.zabbixImageTag }}"
           {{- end }}
           ports:
             - containerPort: 10051
               name: zabbix-server
-              {{- if (default false .Values.zabbixserver.hostPort) }}
+              {{- if (default false .Values.zabbixServer.hostPort) }}
               hostPort: 10051
               {{- end }}
-              {{- if ne "0.0.0.0" .Values.zabbixserver.hostIP }}
-              hostIP: {{ .Values.zabbixserver.hostIP }}
+              {{- if ne "0.0.0.0" .Values.zabbixServer.hostIP }}
+              hostIP: {{ .Values.zabbixServer.hostIP }}
               {{- end}}
             - containerPort: 10052
```

Comment:
I'd remove port 10052 here as anyway nothing is listening on this port. Though it does not hurt to have it here too.